bin/cron/pardot_sync

```diff
@@ -0,0 +1,419 @@
+#!/usr/bin/env ruby
```

Comment:
Does this script assume uniqueness of emails across Code Studio (teacher) accounts? It seems so, something we definitely cannot assume.